src/components/Videos.vue

```diff
@@ -123,6 +124,51 @@ export default {
 			// reset and show poster after play
 			this.$el.autoplay = false
 			this.$el.load()
+		},
+
+		// Save video player's volume and mute status
+		saveVolume() {
+			var videoVolume = {
+				volume: this.$el.volume,
+				muted: this.$el.muted
+			}
+			// store volume status as data of the root component
+			this.$root.$data.videoVolume = videoVolume
+
+			// use localStorage for persistent storage
+			try {
+				localStorage.viewerVideoVolume = JSON.stringify(videoVolume)
+			} catch (e) {
+				console.info('Local storage is not available, cannot save volume settings')
+			}
+		},
+
+		// Restore video player's volume and mute status
+		restoreVolume() {
+			var videoVolume
```

Comment:
This is a local temp variable, so we should use `let`
```suggestion
			let videoVolume
```